src/StreamService/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
```

Comment:
are you sure this file should look like this?

Reply:
fixed